text/0000-north-star.md

```diff
@@ -0,0 +1,466 @@
+  which indicates promising directions for future work.
+- [Production user outreach thread on internals.rust-lang.org]
+  (https://internals.rust-lang.org/t/production-user-research-summary/2530),
+  another strong indicator of Rust's needs.
+- [rust-z]
+  (https://brson.github.io/rust-z),
+  a prototype tool to organize the roadmap.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+The yearly north star RFC could be an unpleasant bikeshed, because it
+simultaneously raises the stakes of discussion while moving away from concrete
+proposals. That said, the *problem* orientation should help facilitate
+discussion, and in any case it's vital to be explicit about our values and
+prioritization.
+
+While part of the aim of this proposal is to increase the effectiveness of our
+team, it also imposes some amount of additional work on everyone. Hopefully the
+benefits will outweigh the costs.
+
+The end-of-year retrospective will require significant effort. It's not clear
+who will be motivated to do it, and at the level of quality it demands. This is
+the piece of the proposal that will probably need the most follow-up work.
+
+# Alternatives
+[alternatives]: #alternatives
+
+Instead of imposing further process structure on teams we might attempt to
+derive a roadmap soley from the data they are currently producing.
+
+To serve the purposes of a 'rallying point', a high-profile deliverable, we
+might release a software product instead of the retrospective. A larger-scope
+product than the existing rustc+cargo pair could accomplish this, i.e.
+[The Rust Platform](http://aturon.github.io/blog/2016/07/27/rust-platform/) idea.
+
+Another rallying point could be a long-term support release.
+
+# Unresolved questions
+[unresolved]: #unresolved-questions
+
+Are 1 year cycles long enough?
+
+Does the yearly report serve the purpose of building anticipation, motivation,
+and creating a compelling PR-bomb?
+
+Is a consistent time-frame for the big cycle really the right thing? One of the
+problems we have right now is that our release cycles are so predictable they
+are almost boring. It could be more exciting to not know exactly when the cycle
+is going to end, to experience the tension of struggling to cross the finish
+line.
+
+How can we account for work that is not part of the planning process
+described here?
+
+How do we address problems that are outside the scope of the standard library
+and compiler itself? (See
+[The Rust Platform](http://aturon.github.io/blog/2016/07/27/rust-platform/) for
+an alternative aimed at this goal.)
+
+How do we motivate the improvement of rust-lang crates and other libraries? Are
+they part of the planning process? The retrospective?
+
+'Problem statement' is not inspiring terminology. We don't want to our roadmap
+to be front-loaded with 'problems'. Likewise, 'goal' and 'retrospective' could
+be more colorful.
+
+Can we call the yearly RFC the 'north start RFC'? Too many concepts?
```

Comment:
On the name itself, I don't think "north star" is the most obvious metaphor, I would prefer a plain name, rather than a metaphorical one.